servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpCookiePair.java

```diff
@@ -138,9 +142,7 @@ public boolean equals(final Object o) {
 
     @Override
     public int hashCode() {
-        int hash = 31 + name.hashCode();
-        hash = 31 * hash + value.hashCode();
-        return hash;
+        return Objects.hash(name, value);
```

Comment:
We tend to prefer the direct calculation vs the helper methods for perf/allocations reasons (ie. array creation for args).